### ISSUE TITLE
The avahi daemon keeps detecting hostname conflicts

### ISSUE THREAD

**Issue by hnez** [linux-automation/meta-lxatac#128]
We have noticed that `avahi-daemon` sometimes produces a lot of log output because it keeps detecting hostname conflicts:

```
Mar 20 11:39:29 lxatac-00015 avahi-daemon[306]: Host name conflict, retrying with lxatac-00015-30
Mar 20 11:39:29 lxatac-00015 avahi-daemon[306]: Registering new address record for 2a0a:edc0:1:2:1a74:e2ff:fea0:113 on tac-bridge.*.
Mar 20 11:39:29 lxatac-00015 avahi-daemon[306]: Registering new address record for 192.168.9.78 on tac-bridge.IPv4.
Mar 20 11:39:29 lxatac-00015 avahi-daemon[306]: Registering new address record for ::1 on lo.*.
Mar 20 11:39:29 lxatac-00015 avahi-daemon[306]: Registering new address record for 127.0.0.1 on lo.IPv4.
Mar 20 11:39:49 lxatac-00015 avahi-daemon[306]: Server startup complete. Host name is lxatac-00015-30.local. Local service cookie is 1771495564.
Mar 20 11:39:49 lxatac-00015 avahi-daemon[306]: Withdrawing address record for 192.168.9.78 on tac-bridge.
Mar 20 11:39:49 lxatac-00015 avahi-daemon[306]: Withdrawing address record for ::1 on lo.
Mar 20 11:39:49 lxatac-00015 avahi-daemon[306]: Withdrawing address record for 127.0.0.1 on lo.
Mar 20 11:39:49 lxatac-00015 avahi-daemon[306]: Host name conflict, retrying with lxatac-00015-31
Mar 20 11:39:49 lxatac-00015 avahi-daemon[306]: Registering new address record for 2a0a:edc0:1:2:1a74:e2ff:fea0:113 on tac-bridge.*.
Mar 20 11:39:49 lxatac-00015 avahi-daemon[306]: Registering new address record for 192.168.9.78 on tac-bridge.IPv4.
Mar 20 11:39:49 lxatac-00015 avahi-daemon[306]: Registering new address record for ::1 on lo.*.
Mar 20 11:39:49 lxatac-00015 avahi-daemon[306]: Registering new address record for 127.0.0.1 on lo.IPv4.
Mar 20 11:40:09 lxatac-00015 avahi-daemon[306]: Server startup complete. Host name is lxatac-00015-31.local. Local service cookie is 1771495564.
Mar 20 11:40:09 lxatac-00015 avahi-daemon[306]: Withdrawing address record for 192.168.9.78 on tac-bridge.
Mar 20 11:40:09 lxatac-00015 avahi-daemon[306]: Withdrawing address record for ::1 on lo.
Mar 20 11:40:09 lxatac-00015 avahi-daemon[306]: Withdrawing address record for 127.0.0.1 on lo.
```

We should find out why that is and if we even need an `avahi-daemon` on the TAC or if it can be replaced by `systemd-resolved`.

**Comment by Emantor** [comment]
This looks to be https://github.com/avahi/avahi/issues/117.

**Comment by hnez** [comment]
We have not observed this since #137 was merged and since that PR is part of the [v24.09](https://github.com/linux-automation/meta-lxatac/releases/tag/v24.09) release now I feel confident in closing this.
